apps/backend/src/lambdas/parser.ts

```diff
@@ -97,60 +156,76 @@ export const handler = async (event?: unknown, _context?: Context) => {
         process.exit(0);
       })();
 
-      const file = await s3.getObject(
+      const file = await s3.getObjectStream(
```

Comment:
Just curious what the difference will be for us by using the getObjectStream vs get Object

Reply:
I need to double check if it works with getObject now, but stream was the easiest way to get the file into a buffer to submit to a post request

Reply:
Yep we want strem in this scenario